.github/CODEOWNERS

```diff
@@ -2,7 +2,10 @@
 /.github/ @Azure/bicep-admins @Azure/avm-core-team-technical-bicep
 /scripts/ @Azure/bicep-admins @Azure/avm-core-team-technical-bicep
 /avm/ @Azure/avm-core-team-technical-bicep
+/avm/ptn/ @Azure/avm-module-reviewers-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/ @Azure/avm-module-reviewers-bicep @Azure/avm-core-team-technical-bicep
 /avm/utilities/ @Azure/avm-core-team-technical-bicep
+/avm/utl/ @Azure/avm-module-reviewers-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Non existing folder. We should either 

1. remove the line and add it once applicable -OR- 
2. comment the line and uncomment it once applicable

Reply:
commented out